src/js/components/Anchor.js

```diff
@@ -41,32 +46,27 @@ export default class Anchor extends Component {
 
     let classes = classnames(
       CLASS_ROOT,
-      this.props.className,
+      className,
```

Comment:
Let's put any provided className prop last in the list. It makes the class easier to read when inspecting in the browser.